test/libsolidity/ASTJSON/userDefinedValueType.json

```
@@ -0,0 +1,264 @@
+{
+  "absolutePath": "a",
+  "exportedSymbols":
+  {
+    "C":
+    [
+      21
+    ],
+    "MyAddress":
+    [
+      2
+    ],
+    "MyUInt":
+    [
+      4
+    ],
+    "f":
+    [
+      16
+    ]
+  },
+  "id": 22,
+  "nodeType": "SourceUnit",
+  "nodes":
+  [
+    {
+      "id": 2,
+      "name": "MyAddress",
+      "nameLocation": "5:9:1",
+      "nodeType": "UserDefinedValueTypeDefinition",
```

```suggestion
      "nodeType": "UserDefinedValueTypeDefinition",
```
As for names: This I would rather have say ``UserDefinedTypeDefinition`` or even just plainly ``TypeDefinition``.